master/internal/rm/tasklist/job.go

```diff
@@ -59,11 +59,11 @@ func JobStats(taskList *TaskList) *jobv1.QueueStats {
 }
 
 // JobStatsByPool returns quick job-related stats about the TaskList, by resource pool.
-func JobStatsByPool(taskList *TaskList, resourcePool string) *jobv1.QueueStats {
+func JobStatsByPool(taskList *TaskList, resourceManager string, resourcePool string) *jobv1.QueueStats {
```

Comment:
there is only one usage of this function and it is in ee

https://github.com/determined-ai/determined-ee/blob/f480795fa1ac7c5d6affc779c8e3f7814e98a34e/master/internal/rm/dispatcherrm/dispatcher_resource_manager.go#L386

I think taskList will only contain tasks of a single resource manager so this parameter is redundant?